src/Symfony/Component/HttpKernel/CacheClearer/Psr6CacheClearer.php

```diff
@@ -25,6 +25,9 @@ public function __construct(array $pools = [])
         $this->pools = $pools;
     }
 
+    /**
+     * @deprecated since version 3.3 and will be removed in 4.0
```

Comment:
the wording of deprecations should always mention the "where": 
`@deprecated since Symfony 3.3 and will be removed in 4.0`

Reply:
Will do